test-site/src/App.tsx

```diff
@@ -12,19 +12,22 @@ const config: HeadlessConfig = {
 
 function App() {
   return (
-    <div className="h-screen w-screen flex justify-center items-center">
+    <div className="h-screen w-screen flex justify-center items-center bg-red-400">
+      {/* <h1 className="external-element">External Element!</h1> */}
       <ChatHeadlessProvider config={config}>
-        <div className="h-5/6 w-1/2">
+        <div className="h-5/6 w-1/2 yext-chat">
           <ChatPanel
             header={
               <ChatHeader title="Clippy's Chatbot" showRestartButton={true} />
             }
           />
         </div>
       </ChatHeadlessProvider>
-      <ChatHeadlessProvider config={config}>
-        <ChatPopUp title="Clippy" />
-      </ChatHeadlessProvider>
+      <div className="yext-chat">
```

Comment:
maybe this is dumb but should we make a ChatStyleScope component which is literally just the div? I feel like it's easier for users to understand an offered component vs "put this random classname on the wrapper div"

Reply:
ooh mm, I think i'm kinda like that. The component can also take care of the import too.

Reply:
maybe `ChatStyleProvider`?